Src/Plugins/Security/SEC101_003.GoogleApiKeyValidator.cs

```diff
@@ -59,7 +59,7 @@ public static string IsValidDynamic(ref string fingerprint, ref string message)
                                                        ref string message)
         {
             const string EnableBilling = "RequestDenied: You must enable Billing on the Google Cloud Project at https://console.cloud.google.com/project/_/billing/enable Learn more at https://developers.google.com/maps/gmp-get-started";
-            const string Deleted = "RequestDenied: This API project was not found. This API project may have been deleted or may not be authorized to use this API. You may need to enable the API under APIs in the console).";
+            const string Deleted = "RequestDenied: This API project was not found. This API project may have been deleted or may not be authorized to use this API. You may need to enable the API under APIs in the console.";
```

Comment:
>Deleted [](start = 25, length = 7)

we could try to include this:
RequestDenied: API keys with referer restrictions cannot be used with this API
RequestDenied: This API project was not found. This API project may have been deleted or may not be authorized to use this API. You may need to enable the API under APIs in the console
RequestDenied: Google has disabled the use of APIs from this API project

and also, we would have to use a contains to validate the following (check ip address):
RequestDenied: This IP, site or mobile application is not authorized to use this API key. Request received from IP address 52.161.72.40, with empty referer

also, this rules does not end with period.